tests/Microsoft.Graph.DotnetCore.Test/Requests/Functional/OneNoteTests.cs

```diff
@@ -27,20 +27,15 @@ public class OneNoteTests : GraphTestBase
             // Get a handle to the first section.
             firstSectionID = sectionPage[0].Id;
         }
-        
-        public async void TestPageCleanUp()
```

Comment:
what is the reason for inlining this?

Reply:
XUnit cleanup is typically performed in Dispose() at the class level and not the test level. This works great for cleaning up local resources but I want to clean up remote resources for only some tests. Since Dispose would be called on all tests (if I understand correctly), we would have attempts to delete remote resources which aren't there. I'm trying to avoid making the unnnecessary calls to delete the remote resources and then handling the errors after a failed attempt. 

Plus, this is faster and these tests are rarely run and are only kept around for convenient code examples at this point.